nixos/lib/systemd-lib.nix

```diff
@@ -235,4 +236,205 @@ in rec {
       ''}
     ''; # */
 
+  makeJobScript = name: text:
```

Comment:
This function can eventually be removed when replaced by https://github.com/NixOS/nixpkgs/pull/154154

Reply:
It's only used in one module, so you could put it in a `let` binding there. (See the other comment about extracting to files)

Reply:
I looked at https://github.com/NixOS/nixpkgs/pull/154154. It looks nice, but it hasn't had activity since januari 7th. I can't wait for that one to be worked on before I can do this PR, which is needed to work on an alternative to #120015 that will be easier to review/merge, because that one is also stalled.